internal/provider/resource_gpg_key.go

```diff
@@ -0,0 +1,279 @@
+func NewGPGKeyResource() resource.Resource {
+	return &gpgKeyResource{}
+}
+
+type gpgKeyResource struct {
+	client *tharsis.Client
+}
+
+// Metadata returns the full name of the resource, including prefix, underscore, instance name.
+func (t *gpgKeyResource) Metadata(ctx context.Context,
+	req resource.MetadataRequest, resp *resource.MetadataResponse) {
+	resp.TypeName = "tharsis_gpg_key"
+}
+
+func (t *gpgKeyResource) Schema(_ context.Context, _ resource.SchemaRequest, resp *resource.SchemaResponse) {
+	description := "Defines and manages a GPG key."
+
+	resp.Schema = schema.Schema{
+		Version:             1,
+		MarkdownDescription: description,
+		Description:         description,
+		Attributes: map[string]schema.Attribute{
+			"id": schema.StringAttribute{
+				MarkdownDescription: "String identifier of the GPG key.",
+				Description:         "String identifier of the GPG key.",
+				Computed:            true,
+				PlanModifiers: []planmodifier.String{
+					stringplanmodifier.UseStateForUnknown(),
+				},
+			},
+			"last_updated": schema.StringAttribute{
+				MarkdownDescription: "Timestamp when this GPG key was most recently updated.",
+				Description:         "Timestamp when this GPG key was most recently updated.",
+				Computed:            true,
+			},
+			"created_by": schema.StringAttribute{
+				MarkdownDescription: "The email address of the user or account that created this GPG key.",
+				Description:         "The email address of the user or account that created this GPG key.",
+				Computed:            true,
+				PlanModifiers: []planmodifier.String{
+					stringplanmodifier.UseStateForUnknown(),
+				},
+			},
+			"ascii_armor": schema.StringAttribute{
+				MarkdownDescription: "The ASCII armored key.",
+				Description:         "The ASCII armored key.",
+				Required:            true,
+				PlanModifiers: []planmodifier.String{
+					stringplanmodifier.RequiresReplace(),
+				},
+			},
+			"fingerprint": schema.StringAttribute{
+				MarkdownDescription: "The fingerprint of the GPG key.",
+				Description:         "The fingerprint of the GPG key.",
+				Computed:            true,
+				PlanModifiers: []planmodifier.String{
+					stringplanmodifier.UseStateForUnknown(),
+				},
+			},
+			"gpg_key_id": schema.StringAttribute{
+				MarkdownDescription: "The GPG key string for this GPG key.",
+				Description:         "The GPG key string for this GPG key.",
+				Computed:            true,
+				PlanModifiers: []planmodifier.String{
+					stringplanmodifier.UseStateForUnknown(),
+				},
+			},
+			"group_path": schema.StringAttribute{
+				MarkdownDescription: "Path of the parent group.",
+				Description:         "Path of the parent group.",
+				Required:            true,
+				PlanModifiers: []planmodifier.String{
+					stringplanmodifier.RequiresReplace(),
+				},
+			},
+			"resource_path": schema.StringAttribute{
+				MarkdownDescription: "Path of the parent group.",
+				Description:         "Path of the parent group.",
```

Comment:
Description for resource_path is incorrect.

Reply:
Should this be, "Path to this GPG Key"? It still refers to the parent group path.

Reply:
Yes.  I didn't notice the addition to the review item.  Will fix.